projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -201,19 +213,53 @@ public function get_link( $url, $text, $title, $query = '', $id = false ) {
 		 *
 		 * @param string $text Sharing service text.
 		 * @param object $this Sharing service properties.
-		 * @param int|false $id Sharing ID.
+		 * @param string|false $id Sharing ID.
 		 * @param array $args Array of sharing service options.
 		 */
 		$text = apply_filters( 'jetpack_sharing_display_text', $text, $this, $id, $args );
 
+		/**
+		 * Filter the sharing data attributes.
+		 *
+		 * @module sharedaddy
+		 *
+		 * @since 10.10.0
```

Comment:
```suggestion
		 * @since $$next-version$$
```

This dynamic value will be automatically updated when we release the next version of Jetpack.